src/Honeybee_infORventPerArea Calculator.py

```diff
@@ -101,7 +101,7 @@ def main(hb_hive, HBZones, airFlowRate, unit):
         warning= None
         infORventPerAreaRes = ''
         try:
-            infORventPerArea.append(standardFlowRate / flrArea)
+            infORventPerArea.append(standardFlowRate)
         except:
             warning = "One of the HBZones did not have any floor area.  The oringal air change values will be kept."
```

Comment:
I think we can just get rid of this try/except loop here since we no longer have to worry about 0 floor area.  Also, I think that this component will need more edits specifically for the workflows related to ACH assigning.

Reply:
I think users might still need this for converting infiltration rates between difference pascals. 

Please merge, since I am not familiar with ACH workflows.